src/python/dxpy/scripts/dx.py

```diff
@@ -174,7 +174,7 @@ class DXCLICompleter():
                    'update': ['stage ', 'workflow ', 'org ', 'member ', 'project '],
                    'org': ['projects ', 'members ']}
 
-    silent_commands = set(['import'])
+    silent_commands = set(['import', 'export'])
```

Comment:
Brett added this set in https://github.com/dnanexus/dx-toolkit/commit/d8408eba59cdbd4b439f01fbbec2c8c933fbcab0 to unadvertise all import commands. Now it unadvertises all export commands as well.